src/classes/HelloDreamforceTest.cls

```diff
@@ -0,0 +1,9 @@
+@isTest
+private class HelloDreamforceTest {
+	
+	@isTest static void test_method_one() {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.